### PR TITLE
Fixed broken image url

### PR DESCRIPTION
**Pull request by sabbatum**

### CODE REVIEW

**Comment by mohanarpit** [comment]
@sabbatum Thanks for opening a PR on Appsmith! 🎉 

Can you please let me know what issue this was causing? Is there a bug that you could link to? I'm asking because I'm not entirely sure why this change is being made and what was broken before this.

File: app/client/cypress/fixtures/viewdsl.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
...
+                  "image": "https://images.pexels.com/photos/60597/dahlia-red-blossom-bloom-60597.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
```
Comment (Nikhil-Nandagopal) [inline app/client/cypress/fixtures/viewdsl.json:75]: @Adityaacharya1807 why was this URL used?

**Comment by hetunandu** [comment]
Hey @sabbatum we had an issue with our CI runs which did not run the tests properly. Can you rebase/merge the latest release branch into your branch to fix this?

**Comment by hetunandu** [comment]
Hey @sabbatum no problem. What we need you to do is update the `release` branch of your [fork](https://github.com/sabbatum/appsmith) so that you get the latest code and fixes. That way this PR will run it's tests again.

Here is a guide you can use for this purpose.
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Let me know if you are still confused and we can probably do this on discord.

**Comment by sabbatum** [comment]
> 
> 
> Hey @sabbatum no problem. What we need you to do is update the `release` branch of your [fork](https://github.com/sabbatum/appsmith) so that you get the latest code and fixes. That way this PR will run it's tests again.
> 
> Here is a guide you can use for this purpose.
> https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
> 
> Let me know if you are still confused and we can probably do this on discord.

Thank you for the guidance. It is very useful. I updated my fork and merged. However, I don't see the test running again.

**Comment by hetunandu** [comment]
@sabbatum Can you join our [discord](https://discord.gg/rBTTVJp) and ping there? I can help you resolve that faster there. You can tag me directly there with the same username @hetunandu

**Comment by sabbatum** [comment]
> 
> 
> @sabbatum Can you join our [discord](https://discord.gg/rBTTVJp) and ping there? I can help you resolve that faster there. You can tag me directly there with the same username @hetunandu

Thank you for all your help, hetunandu. I will not continue with this.

**Pull request closed without merging**